apps/module-presale/truffle.js

```diff
@@ -0,0 +1,6 @@
+const tconfig = require('@aragon/os/truffle-config')
+
+tconfig.solc.optimizer.runs = 3000
```

Comment:
Not sure on the other fundraising apps, but does this not work with 10k (or was there a reason to lower it to 3k)?